lib/src/config/config.rs

```diff
@@ -42,7 +41,7 @@ pub struct Config {
     /// The environment that this configuration corresponds to.
     pub environment: Environment,
     /// The address to serve on.
-    pub address: String,
+    pub address: Address,
```

Comment:
Change address to `Address` seems make `port` obsolete, should we remove it?

Reply:
We need to keep `port` as a separate property because it's very convenient to be able to change the port independently of the address to listen on, especially via environment variables.